src/SamplesApp/SamplesApp.Shared/RoutedEventsPage.xaml

```diff
@@ -0,0 +1,48 @@
+﻿<Page
```

Comment:
This should go in Uno.UI.Samples

Reply:
It's more a development surface than a test control.

Reply:
Alright... but does it belong in the Uno.UI repo?

Reply:
Yes, it should. We'll move it to a "SampleControl" like UI.